single-sign-on/configure/index.md

```diff
@@ -18,7 +18,7 @@ Follow the steps on this page to configure SSO for your organization or company.
 
     >**Note**
     >
-    > Format your domains without protocol or www information, for example, yourcompany.com. This should include all email domains and subdomains users will use to access Docker. Public domains such as gmail.com, outlook.com, etc. aren’t permitted. Also, the email domain should be set as the primary email.
+    > Format your domains without protocol or www information, for example, `yourcompany.com`. This should include all email domains and subdomains users will use to access Docker, for example `yourcompany.com` and `us.yourcompany.com`. Public domains such as `gmail.com`, `outlook.com`, etc. aren’t permitted. Also, the email domain should be set as the primary email.
```

Comment:
It's best to use designated example domains for this (see https://www.rfc-editor.org/rfc/rfc2606.html#section-2), as domains like `yourcompany.com` can be actual domains owned by someone.

For example;

- `yourcompany.example.com`
- `company.example.com`
- `yourcompany.example`

Reply:
Sure, I can make that update.

Reply:
Thanks!

I keep; forgetting to look at adding that to the docs guidelines 😅. In _most_ cases it's not problematic, but we've had some cases where we used an existing domain in the examples, and users (being users) copy/pasted the commands as-is (which isn't, ahem, "ideal" 😅

Reply:
That makes sense!

Reply:
I can add this to our guide and style checkers at some point

Reply:
If you have the time @ChrisChinchilla, that'd be great

Reply:
Yeah was just talking with @thaJeztah about it :)